docs/static/monitoring/monitoring-internal-legacy.asciidoc

```diff
@@ -98,6 +98,14 @@ can specify the location of the PEM encoded certificate with the
 xpack.monitoring.elasticsearch.ssl.certificate_authority: /path/to/ca.crt
 --------------------------------------------------
 
+To add a CA without having it loaded on disk, you can specify a hex-encoded
+SHA 256 fingerprint of the DER-formatted CA with the `ca_trusted_fingerprint` setting:
```

Comment:
To cross-link into the ES docs, you would use `{ref}/configuring-stack-security.html#_use_the_ca_fingerprint_5}`, but I don't think that's the right approach here. 

We're shifting user focus from a Logstash POV and dropping them right into the middle of an multi-step Elasticsearch procedure. We have some of this security information in the Logstash Reference, with a reorg of existing content proposed in #13837. Until we get the stack security story completed (ongoing discussions happening with other teams), I think we should add the CA Fingerprint info to the Logstash Reference rather than linking to it in ES and breaking context for users.  If it were pages of relevant info, I would make a different recommendation.  Let's discuss.

Reply:
@karenzone I've injected an admonition here that copies the Elasticsearch instructions for getting the trusted CA's fingerprint instead of breaking the flow.

Reply:
I love, love, LOVE this approach.  Gets the info to the user without context shifting.  <3